pyveg/coordinates.py

```diff
@@ -154,14 +154,14 @@
 coordinate_store.loc["15"] = {
     "continent": "Africa",
     "country": "Niger",
-    "type": "declining",
+    "type": "labyrinths",
     "latitude": 14.91,
     "longitude": -0.66,
 }
 coordinate_store.loc["16"] = {
     "continent": "Africa",
     "country": "Niger",
-    "type": "declining",
+    "type": "spots/labyrinths",
```

Comment:
Would it be possible to change this to something like "spots-labyrinths" or "spots_labyrinths" ?